### DISCUSSION TITLE
Use bigger character font in some of the labels in Tracking Drives dashboard

### DISCUSSION THREAD

**Discussion by joanquro** [jheredianet/Teslamate-CustomGrafanaDashboards#7] [General]
Hi.

Could you use bigger fonts for the red circled labels in the screenshot, just as it is in the square labels?

Thanks!

![image](https://user-images.githubusercontent.com/62767148/212404359-bc5675be-a73b-441c-8ef7-c20c2f16eb6a.png)

**Comment by jheredianet** [comment]
Hi @joanquro 
How about this?
<img width="1147" alt="td" src="https://user-images.githubusercontent.com/45735191/212436213-2198195e-6ec8-439f-9760-9ea911ecd55e.png">
